doubleml/double_ml.py

```diff
@@ -483,8 +507,10 @@ def fit(self, n_jobs_cv=None, store_predictions=False, store_models=False):
 
                 self._set_score_elements(score_elements, self._i_rep, self._i_treat)
 
+                # calculate rmses and store predictions and targets of the nuisance models
+                self._calc_rmses(preds['predictions'], preds['targets'])
```

Comment:
Hi @SvenKlaassen , 

thanks for this PR! I think adding some diagnostics is really nice.

I think it should be possible to make this a bit more general by using (maybe only a subset of) sklearn's [metrics](https://scikit-learn.org/stable/modules/classes.html#module-sklearn.metrics) either by letting users pass a `callable` for evaluation of the nuisance predictions or by directly supporting the measures. For example, in case $D$ or $Y$ are binary, classification error or cross-entropy loss might be more relevant than the RMSE. What do you think about this?

Reply:
I guess that sklearn's measures have built in some methods to handle exceptions that might occur ...